app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -66,16 +66,14 @@ def ems_inv_to_hashes
       # get_hosts
       get_images
       get_servers
-      get_volumes
-      get_backups
-      get_snapshots
-      get_object_store
```

Comment:
@hsong-rh - Rich commented earlier - this line should not be removed by this PR.